### ISSUE TITLE
nginx not starting due to missing user inside tools_awx_1

### ISSUE THREAD

**Issue by justinotherguy** [ansible/awx#11276]
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](http://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx/issues) for duplicates.
- [X] I understand that AWX is open source software provided for free and that I am not entitled to status updates or other assurances.

### Summary

nginx won't start inside the docker container due to missing user and group creation for nginx

### AWX version

19.3.0

### Installation method

docker on linux

### Modifications

no

### Ansible version

2.7.7

### Operating system

Debian 10.11

### Web browser

Chrome, Safari

### Steps to reproduce

Following [Docker Compose for Development](https://github.com/ansible/awx/blob/devel/tools/docker-compose/README.md) I end up with this:

- awx web ui does not show, when I try to connect via https://localhost:8043/#/home
- from the docker host the container looks fine:
```
5f294a9ee63b   quay.io/awx/awx_devel:HEAD   "/entrypoint.sh laun…"   2 hours ago    Up 2 hours              0.0.0.0:2222->2222/tcp, :::2222->2222/tcp, 0.0.0.0:6899->6899/tcp, :::6899->6899/tcp, 0.0.0.0:7899-7999->7899-7999/tcp, :::7899-7999->7899-7999/tcp, 0.0.0.0:8013->8013/tcp, :::8013->8013/tcp, 0.0.0.0:8043->8043/tcp, :::8043->8043/tcp, 0.0.0.0:8080->8080/tcp, :::8080->8080/tcp, 22/tcp, 0.0.0.0:8888->8888/tcp, :::8888->8888/tcp   tools_awx_1
```
- the port 8043 is also shown as being listened to:
```
# ss -tlnep | grep 8043
LISTEN    0         128                0.0.0.0:8043             0.0.0.0:*        users:(("docker-proxy",pid=935,fd=4)) ino:46990813 sk:18 <->
LISTEN    0         128                   [::]:8043                [::]:*        users:(("docker-proxy",pid=943,fd=4)) ino:46988910 sk:90 v6only:1 <->
```
- trying to access from the docker host via telnet I get a tcp socket connection, but after being established it end immediately:
```
# telnet localhost 8043
Trying ::1...
Connected to localhost.
Escape character is '^]'.
Connection closed by foreign host.
```

### Expected results

- from inside the container things should look like this:
```
bash-4.4# ss -tulpen | grep 8043
tcp   LISTEN 0      128          0.0.0.0:8043       0.0.0.0:*    users:(("nginx",pid=182,fd=7),("nginx",pid=181,fd=7)) ino:47151226 sk:4 <->
bash-4.4#
```
-> nginx is listening to port 8043
- user and group nginx both don't exist inside the container:
```
bash-4.4# grep nginx /etc/passwd /etc/group
/etc/passwd:nginx:x:1000:994:/home/nginx:/bin/bash
/etc/group:nginx:x:994:
bash-4.4#
```

### Actual results

- from inside the container things look different (empty result):
```
bash-4.4# ss -tulpen | grep 8043
bash-4.4#
```
-> nginx is not listening to port 8043
- looking into the nginx log file (/var/log/nginx/error.log) it shows:
```
2021/10/25 08:26:26 [emerg] 163#0: getpwnam("nginx") failed
```
- the user nginx does not seem to exist inside the container:
```
bash-4.4# grep nginx /etc/passwd
bash-4.4#
```
-> this seems to be the problem.

### Additional information

Fix:
add this between lines 243 and 244 of Makefile:
```useradd -g nginx nginx```
so, the whole block reads:
```
nginx:
        useradd -g nginx nginx
        nginx -g "daemon off;"
```

Solution documented by https://github.com/Achim-Hentschel in https://github.com/ansible/awx/issues/9552#issuecomment-816710547

**Comment by justinotherguy** [comment]
update title; if I'm not mistaken, this is not limited to dev setup

**Comment by shanemcd** [comment]
Hello. This was just fixed by #11288. You can cherry-pick that or wait for the next release. Thanks.
